pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/AdopsjonRevurdering.kt

```diff
@@ -33,11 +39,19 @@ object AdopsjonRevurdering : EtterlatteTemplate<BarnepensjonRevurderingAdopsjonD
             )
         }
         outline {
-            includePhrase(Adopsjon.BegrunnelseForVedtaket)
+            includePhrase(
+                Adopsjon.BegrunnelseForVedtaket(
+                    virkningsdato = virkningsdato,
+                    navn = Expression.BinaryInvoke(adoptertAv.fornavn, adoptertAv.etternavn, ConcatMedMellomrom),
```

Comment:
Det finnes en override av `plus` for StringExpression, så du burde kunne skrive `adoptertAv.fornavn + " " + adoptertAv.etternavn`

Reply:
Herleg! Fungerte fint.

Reply:
@routsi Har du ein ide til korleis få inn mellomnamn også her? Det er jo optional (altså String?), så eg hadde sett for meg noko a la
`if (adoptertAv.mellomnavn != null) {
            return adoptertAv.fornavn + " " + adoptertAv.mellomnavn + " " + adoptertAv.etternavn
        }
        return adoptertAv.fornavn+ " " + adoptertAv.etternavn + adoptertAv.etternavn`

Men det kompilerer jo ikkje per no, så llitt annleis må det vel bli.

Reply:
Vi har en optional expression transformasjon: `adoptertAv.mellomnavn.ifNull("")`. Du har også blokk-konstruksjonen `ifNotNull`:
```
ifNotNull(adoptertAv.mellomnavn) { mellomnavn -> text(Bokmal to mellomnavn) }
```
Ikke at jeg anbefaler blokk-konstruksjon i dette tilfellet da.